pkgs/by-name/on/oniux/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+}:
+rustPlatform.buildRustPackage rec {
```

Consider using `finalAttrs` for improved flexibility on `override`

```suggestion
rustPlatform.buildRustPackage (finalAttrs {
```

---

done